example/dryrun/branch_cell.hpp

```diff
@@ -112,7 +112,7 @@ arb::cable_cell branch_cell(arb::cell_gid_type gid, const cell_parameters& param
 
     cell.paint("soma"_lab, "hh");
     cell.paint("dend"_lab, "pas");
-    cell.default_parameters.axial_resistivity = 100; // [Ω·cm]
+    cell.set_default(arb::axial_resistivity{100}); // [Ω·cm]
```

Comment:
Sorry for that!

I have updated the unit tests and python wrapper, but not all the examples (C++ and Python).

This is a premature PR because I want feedback on the changes to the API... I promise to fix the examples.

Reply:
:+1:  I fixed all of the examples